targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -120,6 +123,9 @@ object CoordinatesForm {
             label = nameLabel,
             focus = true,
             loading = state.searching,
+            error = state.searchTerm.isEmpty,
```

Comment:
`|| state.searchError.nonEmpty` ?

Reply:
Actually I only want to show the reddish color when the search term is empty. In case of search errors they will be displayed next to the label

Reply:
👍